pkg/apis/triggers/v1alpha1/cluster_interceptor_types.go

```diff
@@ -112,12 +112,12 @@ func (it *ClusterInterceptor) ResolveAddress() (*apis.URL, error) {
 	if svc == nil {
 		return nil, ErrNilURL
 	}
-	port := defaultPort
+	port := defaultHTTPSPort
 	if svc.Port != nil {
 		port = *svc.Port
 	}
 	url := &apis.URL{
-		Scheme: "http", // TODO: Support HTTPs if caBundle is present
+		Scheme: "https",
```

Comment:
hmm..we'd have to be a bit careful here...if we move this to only support `https`, anyone who has written a cluster interceptor will not be able use Triggers until they upgrade their code. Should we support both for a release or two?

Reply:
Ah i see

Okay so do you mean we need to have support for both HTTP and HTTPS right/

If thats the case we need to take input from deplpyment yaml to know whether they want HTTP or HTTPS
Shall i change implementation that way ?

Reply:
we discussed this in the WG -  we should take the input for whether it is HTTP/HTTPS from interceptor CRD itself.